backend/app/services/share/knowledge_share_service.py

```
@@ -278,103 +327,463 @@ def _on_member_approved(
-                Kind.is_active,
+                Kind.is_active.is_(True),
             )
             .first()
         )
 
-        if not kb:
-            return False, None, False
+        filtered_members = []
+        failed: List[FailedMemberResponse] = []
+        for entry in members_data:
+            target_user_id, role, ent_type, ent_id, ent_display_name = entry
```

Ruff (0.15.12)

[warning] Unpacked variable `target_user_id` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)